core-bundle/src/Resources/contao/elements/ContentAlias.php

```diff
@@ -38,12 +38,30 @@ public function generate()
 			return '';
 		}
 
+		if (is_a($strClass, ContentProxy::class, true))
+		{
+			if (!empty($this->cssID[1]))
+			{
+				$objElement->classes = array_merge((array) $objElement->classes, [$this->cssID[1]]);
+			}
+
+			/** @var ContentProxy $proxy */
+			$proxy = new $strClass($objElement, $this->strColumn);
+
+			if (!empty($this->cssID[0]))
+			{
+				$proxy->addFragmentAttributes(['cssID' => $this->cssID[0]]);
+			}
+
+			return $proxy->generate();
+		}
+
 		$objElement->origId = $objElement->origId ?: $objElement->id;
 		$objElement->id = $this->id;
 		$objElement->typePrefix = 'ce_';
```

Comment:
```suggestion
        else
        {
		    $objElement->origId = $objElement->origId ?: $objElement->id;
     		$objElement->id = $this->id;
	    	$objElement->typePrefix = 'ce_';
	    }
```

Reply:
We may not override the element id for content proxies. Furthermore setting `origId` and `typePrefix` doesn't have any effect for content proxies. Maybe we might set the id of the `ContentAlias` element as fragment attribute as well.